.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 node_modules
 npm-debug.log
+.idea/
```

Comment:
@trotzig could do that, let me give it a look. Most prolly, I'll be needing info/guidance on how to do it. :)